packages/just-task/src/__tests__/resolve.spec.ts

```diff
@@ -186,3 +187,58 @@ describe('resolve', () => {
     expect(resolve('b.txt', 'd')).toContain(path.join('d', 'b.txt'));
   });
 });
+
+describe('resolveConfigFile', () => {
+  afterEach(() => {
+    mockfs.restore();
+    resetResolvePaths();
+  });
+
+  it('default chooses local config', () => {
+    mockfs({
+      config: {
+        'configArgument.ts': 'formConfig',
+        'defaultConfigArgument.ts': 'formDefaultConfig',
+      },
+      'just.config.ts': 'localConfig',
+    });
+    const resolvedConfig = config.resolveConfigFile({ config: undefined, 'default-config': undefined } as any)
```

Comment:
Should `'default-config'` here be `defaultConfig`?

Reply:
@ecraig12345: Thanks! yes it should have been.